docs/pages/development/installation.md

```diff
@@ -8,36 +8,36 @@ import ConfigurationDiff from '~/components/plugins/ConfigurationDiff';
+> Note: These docs assume a basic level of familiarity with React-Native. If you are new to React-Native, please see the [React-Native docs](https://reactnative.dev/docs/getting-started).
 
-If you're just starting your project, you can create a new project from our template with:
+The installation steps on this page are only required to add the `expo-dev-client` library to a React Native or bare project. To add the `expo-dev-client` library to an existing managed project, see our [Getting Started guide](getting-started.md).
 
-<Terminal cmd={["$ npx crna -t with-dev-client"]} />
 
-If you have an existing project, you'll need to [install the package and make a few changes](installation.md) to your **AppDelegate.m**, **MainActivity.java** and **MainApplication.java**.
+If you're just starting your project, you can run the following command to create a new project with Expo modules pre-installed: 
 
-If you created your project with `expo init` or `create-expo-app`, or you already have `expo` and/or other Expo modules up and running, use the tabs marked **With Expo modules**.
+<Terminal cmd={["$ npx crna -y myapp"]} />
 
-If you created your project with `expo init` or `create-expo-app` before SDK 43 and have `react-native-unimodules` up and running, use the tabs marked **With unimodules**. When you're ready, consider upgrading to the newer Expo modules by [following these steps](https://expo.fyi/expo-modules-migration).
+<!-- TODO: Get this to work when you run `eas build` -->
+<!-- If you're just starting your project, you can run the following command to create a new project from our template and then [skip to building](/development/getting-started.md#creating-and-installing-your-first-development-build):
 
-If you created your project with `npx react-native init` and do not have any other Expo packages installed, **starting from version `0.8.0` you'll need to [install Expo modules](../bare/installing-expo-modules)** in order to use `expo-dev-client`.
-
-> **Note**: if you are using _SDK 43 or above_, you'll need to install `expo-dev-client@0.6.0` or above.
+<Terminal cmd={["$ npx crna -t with-dev-client"]} /> -->
 
 ## 1. Installation
 
-Add the `expo-dev-client` library to your package.json.
+If you don't already have `expo` listed as a dependency in your `package.json`, go ahead and **[install Expo modules](../bare/installing-expo-modules)**. Expo modules (`expo`) allows you to seamlessly install any library from the Expo SDK. 
+
+Now, let's go ahead and install `expo-dev-client`.
 
-<InstallSection packageName="expo-development-client" cmd={["npm install expo-dev-client"]} hideBareInstructions />
+<InstallSection packageName="expo-development-client" cmd={["# Navigate to your new app", "cd myapp", "", "# Install expo-dev-client", "expo install expo-dev-client"]} hideBareInstructions />
 
 ### 🍏 iOS
```

Comment:
```suggestion
### iOS
```

Let's remove the emoji 👍 We recently updated some of the guides to remove emojis (WIP)